HEPML.bib

```diff
@@ -3051,7 +3051,7 @@ @article{Monk:2018zsb
 @article{Andreassen:2018apy,
 	Archiveprefix = {arXiv},
 	Author = {Andreassen, Anders and Feige, Ilya and Frye, Christopher and Schwartz, Matthew D.},
-	eprint = {1804.09720},
+	eprint = 1804.09720,
```

Comment:
The `eprint` field need to either be quoted or in `{}` (c.f. latex build failure).